src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -63,6 +64,13 @@ public function ask(InputInterface $input, OutputInterface $output, Question $qu
                 }
             }
 
+            if ($question instanceof ConfirmationQuestion) {
+                $value = trim(fgets(STDIN));
```

Comment:
It should use `$input->getStream()` and `STDIN` as fallback, which implies to move up the intialization of `$this->inputStream` below